backend/utils/plugins.py

```
@@ -172,14 +176,16 @@ def _single(plugin: Plugin):
 
     [t.start() for t in threads]
     [t.join() for t in threads]
+
     messages = []
     for key, message in results.items():
         if not message:
             continue
-        messages.append(add_plugin_message(message, key, uid))
+        messages.append(add_plugin_message(message, key, uid, memory.id))
```

Ruff

Undefined name `memory`

(F821)